apiserver/facade/interface.go

```diff
@@ -292,10 +297,6 @@ type Authorizer interface {
 	// EntityHasPermission reports whether the given access is allowed for the given
 	// target by the given entity.
 	EntityHasPermission(ctx context.Context, entity names.Tag, operation permission.Access, target names.Tag) error
-
-	// ConnectedModel returns the UUID of the model to which the API
-	// connection was made.
-	ConnectedModel() string
```

Comment:
There are still some implementations of ConnectedModel in the codebase, you can find them with grep, these should go as well

Reply:
Ah, I missed this one.